chef-config/lib/chef-config/config.rb

```diff
@@ -151,6 +151,16 @@ def self.derive_path_from_chef_repo_path(child_path)
     # Defaults to <chef_repo_path>/acls.
     default(:client_path) { derive_path_from_chef_repo_path('clients') }
 
+    # Location of containers on disk. String or array of strings.
+    # Defaults to <chef_repo_path>/containers.
+    # Only applies to Enterprise Chef commands.
+    default(:container_path) { derive_path_from_chef_repo_path('containers') }
+
+    # Location of cookbook_artifacts on disk. String or array of strings.
+    # Defaults to <chef_repo_path>/cookbook_artifacts.
+    # Only applies to Enterprise Chef commands.
```

Comment:
I guess the whole "enterprise" vs. not is kinda moot now, but Cookbook artifacts are objects that aren't related to the authz system so I don't think this is accurate anyway?